### DISCUSSION TITLE
Adding repos to a Podman / Docker built Void VM

### DISCUSSION THREAD

**Discussion by philiprhoades** [void-linux/void-packages#53018] [General]
People,

I found Void because I was looking for a non-systemd, lightweight Linux (I have been using mostly RH / Fedora for decades).

I built this Void VM and it is working but now I want to install some more stuff via the package manager - however all the repo stuff is missing - I found this page but that doesn't help:

https://docs.voidlinux.org/xbps/repositories/mirrors/changing.html?highlight=Repositories#changing-mirrors

Has someone got a repo file that I can just create manually?

Thanks! (and thanks to the Void developers - this explorations should be interesting!)

**Comment by classabbyamp** [comment]
what does xbos-query -L show?
